src/Controller/PartyController.php

```diff
@@ -33,14 +33,25 @@ public function __construct(PartyHelper $partyHelper)
     /**
      * @Route("/add", name="party_add")
      */
-    public function addParty(CaseEntity $case, Request $request): Response
+    public function addParty(CaseEntity $case, PartyHelper $partyHelper, Request $request): Response
     {
         $this->denyAccessUnlessGranted('edit', $case);
 
         $party = new Party();
 
+        $preSelectType = $request->get('type');
+
+        $counterPartyTypes = $partyHelper->getCounterpartyTypesByCase($case);
+        // We use the assumption that first mentioned party on list of parties is the 'main' party type e.g. lejer/udlejer.
+        $preSelectValue =
+            'counterparty' === $preSelectType
+                ? reset($counterPartyTypes)
```

Comment:
`reset` returns `false` for an empty array. You probably want to convert this to `null`.

Reply:
Hopefully this is never the case (empty), as party and counter party types are required upon creating boards.

Changed `reset($counterPartyTypes)` to `reset($counterPartyTypes) ?: null`.